sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/OpenTelemetry.Exporter.AzureMonitor.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
+    <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
+    <Version>1.0.0-preview.0</Version>
```

Comment:
Do we want this to be `1.0.0` or `0.0.1` 😄

Reply:
https://semver.org/#how-do-i-know-when-to-release-100